src/engines/transit_types.ts

```diff
@@ -4,6 +4,10 @@ export type ITransitBatchPlaintext = Array<{
     plaintext: string;
     context?: string;
 }>;
+export type ITransitRawBatchPlaintext = Array<{
+    plaintext?: string;
```

Comment:
Maybe add a link to the respective Vault issue, to explain why this curious typing is necessary?

Reply:
Oh, nevermind. Just noticed that there's a reference to that issue elsewhere.

Reply:
👍